micro-rdk-server/Cargo.toml

```diff
@@ -50,8 +50,8 @@ path = "esp32/main.rs"
 
 
 [package.metadata.esp-idf-sys]
-esp_idf_tools_install_dir = "global"
+esp_idf_version = "v4.4.4"
+esp_idf_tools_install_dir = "custom:target/.embuild/espressif"
```

Comment:
@viamrobotics/micro-rdk-team - Small update here: this will now make the esp-idf environment get installed under the `target` directory by default. You can override this on the command line, if you want, by setting `ESP_IDF_TOOLS_INSTALL_DIR=global`, or to any of the other valid values. I like this better because it now means that `cargo clean` will also destroy any cached esp-idf installations, making it easier to really start "clean". If esp-idf installations are going to `~/.espressif` by default it is too easy to be unaware that you are relying on local state.